src/uct/ib/rc/accel/rc_mlx5_devx.c

```diff
@@ -241,7 +238,7 @@ uct_rc_mlx5_devx_init_rx_tm(uct_rc_mlx5_iface_common_t *iface,
     UCT_IB_MLX5DV_SET(xrqc, xrqc, tag_matching_topology_context.log_matching_list_sz,
                                   ucs_ilog2(iface->tm.num_tags) + 1);
     UCT_IB_MLX5DV_SET(xrqc, xrqc, dc,       dc);
-    UCT_IB_MLX5DV_SET(xrqc, xrqc, cqn,      dvcq.cqn);
+    UCT_IB_MLX5DV_SET(xrqc, xrqc, cqn,      iface->cq[UCT_IB_DIR_RX].cq_num);
```

Comment:
Minor: imo, this alignment looks weird (laso it is broken by the line 238). maybe have just one space here and above?

Reply:
It is aligned for the 4th argument and it's broken on 238 just because this row has too long 3rd argument. Anyway if you do not object I don't want to change the existing alignment at least as part of this PR.

Reply:
ok